src/utilities/validation.ts

```diff
@@ -268,4 +269,10 @@ export const isHandle: ValidationMethodFactory = property => value => {
   }
 
   return `${property} must only contain lowercase letters, numbers, and dashes`
-}
\ No newline at end of file
+}
+
+const SNAKE_CASE_REGEX = /^[a-z0-9]+(?:_[a-z0-9]+)*$/
+
+export const isSnakeCase = (message?: string): ValidationRule<unknown> => (value) => {
```

Comment:
So part of the reason for validation methods is they can return a string error message. Do we really need to to be customizable? I'd expect this to return the proper error message.

Reply:
The second argument that's passed from the validation is the name of the field.